en/uavcan/bootloader_installation.md

```diff
@@ -1,118 +0,0 @@
-# UAVCAN Bootloader Installation
```

Comment:
Why is this deleted/where did the information go?

Reply:
I decided listing multiple ways to flash bootloaders via SWD is out of scope for the PX4 docs, there's tons of materials on the web on how to do it for various devices, better than we could write. However, see cannode_fw.md#dronecan-bootloader-information, where I wrote a basic summary.

Reply:
@coderkalyan I agree we shouldn't list the ways, but I do think that providing links to the instructions would be useful. In "PX4 DroneCAN Firmware" you link to the main product guides for the different ways of doing this - but if there is a more specific link for installing it would be good to use that.

Further, this page was a generic bootloader page, so this is where you'd come for instructions on any firmware. I am not sure someone would expect to find the information for random CAN firmware in "PX4 DroneCAN Firmware".

In other words, this is a question of discoverability. 

"Somewhere" we should say that the installation is covered by the manufacturer for the different probes, and that for dronecan you can find the information at Xxxx. That might be in the top level CAN page?